src/App.jsx

```diff
@@ -1,7 +1,63 @@
+import { useState } from "react";
+import QUESTIONS from "./quizQuestions";
+import QuestionForm from "./components/QuestionForm";
+import ResultForm from "./components/ResultForm";
+import Button from "./components/Button";
+import QuizTime from "./components/QuizTime";
 import "./App.css";
 
+const userAnswers = [];
+
 function App() {
-  return <>Hello World</>;
+  const [question, setQuestion] = useState(-1);
```

Comment:
You initialize the question state to -1 to represent the "start" of the quiz before any questions are shown. You then increment this index for each question. Consider renaming this variable for clarity, e.g., currentQuestionIndex.